client/asset/eth/eth.go

```diff
@@ -2083,6 +2116,8 @@ func (w *assetWallet) Redeem(form *asset.RedeemForm, feeWallet *assetWallet, non
 	}
 
 	txHash := tx.Hash()
+	w.addPendingTx(w.assetID, txHash, 0, redeemedValue, originalFundsReserved)
```

Comment:
Shouldn't `gasFeeCap * gasLimit` be used here ? Since `originalFundsReserved` isn't necessarily what's used all the time.